back-end/controllers/user.js

```diff
@@ -3,22 +3,16 @@ const router = express.Router();
 const { queryDB } = require("../helpers/db-helpers");
 
 router.get("/user/:userId", async function (req, res, next) {
-  // will return {'username': 'sample', 'secret': 'sample} for given userid
   try {
     const userId = req.params["userId"];
 
-    const query = `SELECT secret
+    const query = `SELECT secret_id, secret
       FROM Users u INNER JOIN Secrets s
       ON u.user_id = s.user
       WHERE u.user_id = ${userId}`;
 
     const results = await queryDB(query);
-
-    if (results.length <= 0) {
-      return res.status(401).send("Not found.");
-    } else {
-      return res.status(200).send(results);
-    }
+    res.status(200).send(results);
```

Comment:
Just a little confused on this line - shouldn't it have a `return` to be returning the response?

Reply:
So I just looked into this as well, but it seems it's okay, BUT you should have a `return` statement if you have conditional logic ([source](https://stackoverflow.com/questions/52919585/node-js-return-res-status-vs-res-status)).